sdks/framework/typescript/workflow.mdx

```diff
@@ -51,10 +54,16 @@ workflow(
 <ParamField path="options" type="WorkflowOptions">
     An optional options object for workflow level configurations
 
-    <Expandable title="properties">
-        <ParamField path="payloadSchema" type="JSON Schema | Zod">
+    <Expandable title="properties" defaultOpen>
```

Comment:
Most folks will like to use the options, so I specified the `<Expandable/>` to be open by default.